### DISCUSSION TITLE
LinkedIn profile facility

### DISCUSSION THREAD

**Discussion by pravee42** [community/community#36957] [Other Feature Feedback, Questions, & Ideas]
Why does GitHub not have a LinkedIn profile option in GitHub profile with only having the option to add twitter profile.
It would be great if it had one for those who are not adding their LinkedIn profile in the readme page

**Comment by ossanna16** [comment]
Hi there @pravee42 and welcome to our community 😄. Thanks for the suggestion!

**Reply by pravee42** [reply]
Ok @ossanna16

**Comment by DSDmark** [comment]
Hi @pravee42,

Ya, **LinkedIn** is cool. But I want to add my **Stack Overflow** profile. 

Why the way, that discussion is already running on community.

**Reply by pravee42** [reply]
Yeah that's cool too

**Comment by jfullstackdev** [comment]
well, if you have your landing README page, you can add as many links as you want other than those provided by GitHub, like mine 

![image](https://user-images.githubusercontent.com/47092464/204702094-b757c7bc-95ba-4d22-b623-53deedb23d31.png)

even extra pins for repos can be added

another side information, I maintain my GitHub Profile for the sake of using it as my credentials and lately landing a real job in development even if I'm not a graduate of IT or Computer Science, I just started building personal projects and use it for the demonstration of my would be actual coding job

**Reply by pravee42** [reply]
That is an cool idea but if you see the github you can notice that the they option to add twitter account.
Those who want to automatically merge their skills in github and linked in ass so they need not want to update the skills and role in the github readme as well only they can update in the the linked in profile itself
